### ISSUE TITLE
1.7.10 server- refuses to load dropboxlink

### ISSUE THREAD

**Issue by Golummechanic** [aidancbrady/Mekanism#5086]
#### Issue description:
My server refuses to load mekanisums for 1.7.10
the error message reads that it failed to connect to a dropboxlink:
https://dl.dropbox.com/u/90411166/Donators/Mekanism.txt

Im running this whole ordeal on a Apex server with a bunch of other mods, i have no difficulties in singleplayer but on a server it just refuses to load up.

#### Steps to reproduce:
1.get a test server (apex if you can)
2.install forge 1.7.10 and mekanism core and generators
3.read the concel while it starts the server, the server will lag out and you can read a error message that it failed to connect to the dropbox file

#### Version (make sure you are on the latest version before reporting):
**Forge:*forge 1.7.10* (not able to get a version number from the concel or files) 
**Mekanism:*1.7.10-9.1.0.281* 

#### If a (crash)log is relevant for this issue, link it here: (_It's almost always relevant_)
-No crashlog was given sadly i had to manually shot down the server

**Comment by thommy101** [comment]
I don't think the server actually hangs on a failed Dropbox connection, but, there has been a new 1.7.10 release some time ago to move the links from Dropbox to Aidans own site.
It's available on curseforge

**Comment by Golummechanic** [comment]
Thanks for the quick reply,
It was the only issue i couod find while reading the concel. I shall check tomorow if this fixes the problem.

could you explain why this txtfile has to be downloaded externally and not just include it in the mod itself?

**Comment by thiakil** [comment]
That would require a new release every time aidan changed the file.

**Comment by Golummechanic** [comment]
thanks for the explination thiakil ^^

I have downloaded the updated version on curseforge and injected them in my server.
it frose up again but after some log reading i found that the server was waiting for a responce from me.
I do not know if it did that too, but it seems the server is up and running.

[Server console report.txt](https://github.com/aidancbrady/Mekanism/files/2101004/Server.console.report.txt)
I included the console report for closer inspection if nessesary.

I can't say for a 100% if my server is working as my worklaptop isnt able to run the modpack, if I encounter any problems i be sure to report. if not then I will just messages a quick thank you.
